gatsby-config.ts

```diff
@@ -48,7 +48,17 @@ const config: GatsbyConfig = {
         siteUrl, // Update with your website URL
         trailingSlash: 'always',
         i18nextOptions: {
-          ns: ['index', 'blog', 'book', 'home'],
+          defaultNS: 'index',
```

Comment:
Should I update it each time new locale file is created?

Reply:
Yes. I will add an automation script for this task. . It is recommended to explicitly specify the namespaces (ns) in the i18nextOptions configuration to load the required namespaces for each locale. This ensures that the translations are organized and properly loaded.

Reply:
@kaleemullahdev So manually or via script?)) It's not clear what the final decision will be.

Reply:
@romanr111 I have added a script that automatically picks the namespace from the locales folder, eliminating the need to manually add the namespace here after adding a new JSON file.